github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1798,6 +1799,7 @@ presubmits:
       nodeSelector:
         type: bare-metal-external
   - always_run: false
+    optional: true
```

Comment:
@brianmcarey 
please note i changed `pull-kubevirt-e2e-k8s-1.27-sig-compute-realtime`
so it wont be considered phase 2